src/bin/pg_autoctl/cli_common.c

```diff
@@ -809,7 +792,7 @@ void
 set_first_pgctl(PostgresSetup *pgSetup)
 {
 	char **pg_ctls = NULL;
-	int n = search_pathlist(getenv("PATH"), "pg_ctl", &pg_ctls);
+	int n = search_path("pg_ctl", &pg_ctls);
```

Comment:
Multiple entries are handled differently everywhere:
- Error out that there are multiple `pg_ctl` 
- Take the first one

Since grabbing the first one is very simple, i.e. `results[0]` I don't think there's much advantage to to this.

Reply:
Actually I did this anyway for the last case. But I kept the old function as well for more advanced handling.